consensus/spos/bls/subroundStartRound.go

```diff
@@ -44,6 +46,7 @@ func NewSubroundStartRound(
 		executeStoredMessages:         executeStoredMessages,
 		resetConsensusMessages:        resetConsensusMessages,
 		outportHandler:                outport.NewNilOutport(),
+		mutex:                         sync.RWMutex{},
```

Comment:
Kind of tricky. If this mutex is only used for setting the outport handler, please rename to `outportMutex`, perhaps - since here we are in consensus components?

Reply:
Also, it can be `Mutex` (instead of (`RWMutex`).

Reply:
Yes, that is a better naming. outportMut or mutOutport or outportMutex as to no think that mutex is used in some mission-critical way.

Reply:
done.